### PR TITLE
add(mc): new applications-veeam-wsman MC

### PR DESCRIPTION
**Pull request by omercier**
## Description

New MC

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+2. Renseignez les macros désirées (par exemple, ajustez les seuils d'alerte). Les macros indiquées ci-dessous comme requises (**Obligatoire**) doivent être renseignées.
+
+<Tabs groupId="sync">
+<TabItem value="Job-Status" label="Job-Status">
+
+| Macro           | Description                                                                                                                                                                                       | Valeur par défaut                                   | Obligatoire |
+|:----------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------|:-----------:|
+| FILTERENDTIME   | Filter job with end time greater than current time less value in seconds (Default: 86400)                                                                                                         | 86400                                               |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md:212]: less?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+| FILTERNAME      | Filter job name (can be a regexp)                                                                                                                                                                 |                                                     |             |
+| FILTERSTARTTIME | Filter job with start time greater than current time less value in seconds                                                                                                                        |                                                     |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md:214]: less

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md [hunk 3]
```diff
@@ -0,0 +1,510 @@
...
+| FILTERCOUNTERS  |                                                                                                                                                                                                   |                                                     |             |
+| OKSTATUS        | Set ok threshold for status. Can used special variables like: %{display}, %{status}, %{type}, %{is\_running}, %{scheduled}                                                                        |                                                     |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md]:
```suggestion
| OKSTATUS        | Set OK threshold for status. Can use special variables like: %{display}, %{status}, %{type}, %{is\_running}, %{scheduled}                                                                        |                                                     |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md [hunk 4]
```diff
@@ -0,0 +1,510 @@
...
+| WARNINGLONG     | Set warning threshold for long jobs. Can used special variables like: %{display}, %{status}, %{type}, %{elapsed}                                                                                  |                                                     |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md]:
```suggestion
| WARNINGLONG     | Set warning threshold for long jobs. Can use special variables like: %{display}, %{status}, %{type}, %{elapsed}                                                                                  |                                                     |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md [hunk 5]
```diff
@@ -0,0 +1,510 @@
...
+| CRITICALLONG    | Set critical threshold for long jobs. Can used special variables like: %{display}, %{status}, %{type}, %{elapsed}                                                                                 |                                                     |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md]:
```suggestion
| CRITICALLONG    | Set critical threshold for long jobs. Can use special variables like: %{display}, %{status}, %{type}, %{elapsed}                                                                                 |                                                     |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md [hunk 6]
```diff
@@ -0,0 +1,510 @@
...
+| CRITICALSTATUS  | Set critical threshold for status (Default: '%{is\_running} == 0 and not %{status} =~ /Success/i'). Can used special variables like: %{display}, %{status}, %{type}, %{is\_running}, %{scheduled} | %{is\_running} == 0 and not %{status} =~ /Success/i |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md]:
```suggestion
| CRITICALSTATUS  | Set critical threshold for status (Default: '%{is\_running} == 0 and not %{status} =~ /Success/i'). Can use special variables like: %{display}, %{status}, %{type}, %{is\_running}, %{scheduled} | %{is\_running} == 0 and not %{status} =~ /Success/i |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md [hunk 7]
```diff
@@ -0,0 +1,510 @@
...
+| WARNINGSTATUS   | Set warning threshold for status. Can used special variables like: %{display}, %{status}, %{type}, %{is\_running}, %{scheduled}                                                                   |                                                     |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md]:
```suggestion
| WARNINGSTATUS   | Set warning threshold for status. Can use special variables like: %{display}, %{status}, %{type}, %{is\_running}, %{scheduled}                                                                   |                                                     |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md [hunk 8]
```diff
@@ -0,0 +1,510 @@
...
+|:----------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------|:---------------------------------|:-----------:|
+| FILTERTO                          | Filter licenses by person/organization (can be a regexp)                                                                                      |                                  |             |
+| FILTERTYPE                        | Filter licenses by type (can be a regexp)                                                                                                     |                                  |             |
+| FILTERSTATUS                      | Filter licenses by status (can be a regexp)                                                                                                   |                                  |             |
+| UNIT                              | Select the unit for expires threshold. May be 's' for seconds,'m' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is seconds |                                  |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-wsman.md:231]: expired thresholds?

File: pp/integrations/plugin-packs/procedures/applications-veeam-wsman.md [hunk 9]
```diff
@@ -0,0 +1,509 @@
...
+
+<Tabs groupId="sync">
+<TabItem value="App-Veeam-WSMAN-custom" label="App-Veeam-WSMAN-custom">
+
+| Service Alias | Service Template                    | Service Description                     |
+|:--------------|:------------------------------------|:----------------------------------------|
+| Job-Status    | App-Veeam-Job-Status-WSMAN-custom   | Monitor the statuses of the backup jobs |
+| Licenses      | App-Veeam-Licenses-WSMAN-custom     | Monitor licenses                        |
+| Repositories  | App-Veeam-Repositories-WSMAN-custom | Monitor backup repositories             |
+| Tape-Jobs     | App-Veeam-Tape-Jobs-WSMAN-custom    | Monitor job tape status                 |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-veeam-wsman.md:26]:
```suggestion
| Tape-Jobs     | App-Veeam-Tape-Jobs-WSMAN-custom    | Monitor the status of tape jobs             |
```

File: pp/integrations/plugin-packs/procedures/applications-veeam-wsman.md [hunk 10]
```diff
@@ -0,0 +1,509 @@
...
+2. Fill in the macros you want (e.g. to change the thresholds for the alerts). Some macros are mandatory (see the table below).
+
+<Tabs groupId="sync">
+<TabItem value="Job-Status" label="Job-Status">
+
+| Macro           | Description                                                                                                                                                                                       | Default value                                       | Mandatory   |
+|:----------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------|:-----------:|
+| FILTERENDTIME   | Filter job with end time greater than current time less value in seconds (Default: 86400)                                                                                                         | 86400                                               |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-veeam-wsman.md:213]: less?

File: pp/integrations/plugin-packs/procedures/applications-veeam-wsman.md [hunk 11]
```diff
@@ -0,0 +1,509 @@
...
+| FILTERNAME      | Filter job name (can be a regexp)                                                                                                                                                                 |                                                     |             |
+| FILTERSTARTTIME | Filter job with start time greater than current time less value in seconds                                                                                                                        |                                                     |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-veeam-wsman.md:215]: less?

File: pp/integrations/plugin-packs/procedures/applications-veeam-wsman.md [hunk 12]
```diff
@@ -0,0 +1,509 @@
...
+|:----------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------|:---------------------------------|:-----------:|
+| FILTERTO                          | Filter licenses by person/organization (can be a regexp)                                                                                      |                                  |             |
+| FILTERTYPE                        | Filter licenses by type (can be a regexp)                                                                                                     |                                  |             |
+| FILTERSTATUS                      | Filter licenses by status (can be a regexp)                                                                                                   |                                  |             |
+| UNIT                              | Select the unit for expires threshold. May be 's' for seconds,'m' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is seconds |                                  |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-veeam-wsman.md:232]:
```suggestion
| UNIT                              | Select the unit for expired thresholds. May be 's' for seconds,'m' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is seconds |                                  |             |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by lucie-dubrunfaut — APPROVED** [review]

**Pull request merged**